src/main/resources/assets/sass/_fonts.scss

```diff
@@ -0,0 +1,25 @@
+@font-face {
```

Comment:
I can include the `fonts.css` from `govuk_template` inside the head to DRY this up